acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/configuration/BesuNodeConfigurationBuilder.java

```diff
@@ -103,11 +105,10 @@ public BesuNodeConfigurationBuilder enablePrivateTransactions() {
     return this;
   }
 
-  public BesuNodeConfigurationBuilder jsonRpcAuthenticationEnabled() throws URISyntaxException {
+  public BesuNodeConfigurationBuilder jsonRpcAuthenticationEnabled(final String authFile)
```

Comment:
rename the function to imply the authFile is coming from SystemResources (also - this is no longer an enable/disable function